news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,101 @@
+
+Our target is to be able to publish a release every 6–9 months. Since every other release will be a major version, the community will be able to expect a new major version every 12 to 18 months.
+
+Now let's talk about support. Currently, patch versions are released at most 6 weeks counting from the moment the maintainers decided that a given bug justifies a patch release. Since patches are only published for the latest version, this effectively means that support of a given PrestaShop minor version generally ends the moment the following minor version is published (e.g. support for 1.7.7.x version ended when 1.7.8.0 was released). This will continue to be true with the new release roadmap.
+
+However, we understand that upgrading to a major version will be harder than upgrading to a minor or patch version, and that people might choose not to upgrade to a new major version as soon as it's released. Because of this, we decided to provide extended critical and security bug fixes for minor versions, as necessary, for up to 12 months after the following major version has been released.
+
+Here is what our ideal release & support roadmap should look like:
+
+![Ideal roadmap](/assets/images/2021/12/roadmap.png)
+
+To recap:
+
+- A release every 6~9 months
+- Alternating minor and major versions
+- Patches when necessary between releases
+- Up to 1 year of additional critical fixes for minor versions
+
+## Say hello to PrestaShop 8
+
+Our next release, **PrestaShop 8**, will mark the simplification of the project's version numbering scheme. The "1" that preceded previous versions, a heritage that didn't carry any semantic meaning, will no longer be included in future releases.
+
+![New version scheme](/assets/images/2021/12/v8-scheme.png)
+
+
+Adopting the industry standard 3-group version scheme will allow to simplify the way PrestaShop versions are identified, as well as enhance interoperability with third-party software, like Composer, who depend on standardized versions.
+
+### An evolutionary, not revolutionary, release
+
+As explained before, the SemVer convention defines a version in terms of backward compatibility, not "big" or "small" changes. This means that a major version doesn't necessarily introduce big, revolutionary changes. It just means that _some_ things won't work the exact same way as before, that's all.
+
+In that sense, PrestaShop 8 will continue and accelerate the transformative work towards the Future Architecture started in 1.7, but with a more pragmatic and focused approach (which will be covered in a dedicated article).
+
+This new version will focus on two main initiatives:
+
+##### 1. Technology update & cleanup
```

Comment:
If you use "View file" you'll see this displays weird 🤔 

Semantic is good, how github markdown displays it seems off

Reply:
Don't worry, it displays well in Jekyll (I tested it)